src/index.html

```diff
@@ -10,7 +11,46 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="search-form"
+      data-qa="big"
+    >
+
+      <label
+        for="#form-input-big"
+        class="search-form__label search-form__label__big"
+      >
+        <input
+        type="text"
+        id="form-input-big"
```

Comment:
Use camelCase for values of name attribute - they should be valid as JavaScript object keys. It should not contain spaces, “-” or other special characters.